daemon/cmd/cni/cell.go

```diff
@@ -69,6 +75,7 @@ func enableConfigManager(lc hive.Lifecycle, log logrus.FieldLogger, cfg Config,
 func newConfigManager(log logrus.FieldLogger, cfg Config, debug bool) *cniConfigManager {
 	if cfg.CNIChainingMode == "aws-cni" && cfg.CNIChainingTarget == "" {
 		cfg.CNIChainingTarget = "aws-cni"
+		cfg.CNIExternalRouting = true
```

Comment:
@aditighag the nice thing with this particular approach is that for aws-cni we don't need to document anything, it should just autodetect and work due to this line. The existing instructions already configure this option.

Arguably we could document the option further for the general case, but I think that for now the cmdref should be good enough. We can keep an eye out for any future other external routing agents and point this out if it becomes relevant.

Reply:
Sgtm! 👍